test/fluree/db/query/sparql_test.cljc

```diff
@@ -9,587 +9,602 @@
+       (testing "MAX"
+         (let [query "SELECT ?fullName (MAX(?favNums) AS ?max)
                    WHERE {?person person:favNums ?favNums.
                           ?person person:fullName ?fullName}"
-            {:keys [select]} (sparql/->fql query)]
-        (is (= ["?fullName" "(as (max ?favNums) ?max)"]
-               select))))
-    (testing "MIN"
-      (let [query "SELECT ?fullName (MIN(?favNums) AS ?min)
+               {:keys [select]} (sparql/->fql query)]
+           (is (= ["?fullName" "(as (max ?favNums) ?max)"]
+                  select))))
+       (testing "MIN"
+         (let [query "SELECT ?fullName (MIN(?favNums) AS ?min)
                    WHERE {?person person:favNums ?favNums.
                           ?person person:fullName ?fullName}"
-            {:keys [select]} (sparql/->fql query)]
-        (is (= ["?fullName" "(as (min ?favNums) ?min)"]
-               select))))
-    (testing "SAMPLE"
-      (let [query "SELECT ?fullName (SAMPLE(?favNums) AS ?sample)
+               {:keys [select]} (sparql/->fql query)]
+           (is (= ["?fullName" "(as (min ?favNums) ?min)"]
+                  select))))
+       (testing "SAMPLE"
+         (let [query "SELECT ?fullName (SAMPLE(?favNums) AS ?sample)
                    WHERE {?person person:favNums ?favNums.
                           ?person person:fullName ?fullName}"
-            {:keys [select]} (sparql/->fql query)]
-        (is (= ["?fullName" "(as (sample1 ?favNums) ?sample)"]
-               select))))
-    (testing "SUM"
-      (let [query "SELECT ?fullName (SUM(?favNums) AS ?sum)
+               {:keys [select]} (sparql/->fql query)]
+           (is (= ["?fullName" "(as (sample1 ?favNums) ?sample)"]
+                  select))))
+       (testing "SUM"
+         (let [query "SELECT ?fullName (SUM(?favNums) AS ?sum)
                    WHERE {?person person:favNums ?favNums.
                           ?person person:fullName ?fullName}"
-            {:keys [select]} (sparql/->fql query)]
-        (is (= ["?fullName" "(as (sum ?favNums) ?sum)"]
-               select))))
-    ;;TODO: not yet supported
-    #_(testing "GROUP_CONCAT")))
+               {:keys [select]} (sparql/->fql query)]
+           (is (= ["?fullName" "(as (sum ?favNums) ?sum)"]
+                  select))))
+       ;;TODO: not yet supported
+       #_(testing "GROUP_CONCAT"))))
 
-(deftest parse-where
-  (testing "simple triple"
-    (let [query "SELECT ?person
+;; TODO: Delete this :clj wrapper once these pass
+#?(:clj (deftest ^:pending parse-where
+          (testing "simple triple"
+            (let [query "SELECT ?person
                  WHERE {?person person:handle \"jdoe\".}"
-          {:keys [where]} (sparql/->fql query)]
-      (is (= [["?person" "person:handle" "jdoe"]]
-             where))))
-  (testing "multi clause"
-    (let [query "SELECT ?person ?nums
+                  {:keys [where]} (sparql/->fql query)]
+              (is (= '[{"@id"           "?person"
```

Comment:
Super-trivial, but I don't think you need the quote here

Reply:
Oh, nope. At one point I thought it was going to output symbols now, but it doesn't. Thanks!